content/en/tools/digital-currencies/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+date: '2023-03-21'
+section_menu: tools
+menu:
+  tools:
+    name: Digital Currencies
+    weight: 600
+    identifier: digital-currencies
+title: Welcome to Digital Currencies
+description: "Documentation describing the Digital Currencies feature from R3"
+
+---
+
+# Welcome to Digital Currencies
+
+Welcome to the documentation for the Digital Currencies for Corda 4 GUI. The purpose of this documentation is to describe the concepts behind Digital Currencies, show the lifecycle of a digital token definition, describe the functionality available to participants in a digital currency network, and how these action can be performed via the Digital Currencies GUI.
+
+R3 Digital Currencies for Corda 5 enables central banks, commercial banks and institutions to define their own digital currency token types and perform the following actions based on those definitions:
+
+* mint tokens
+* request an issuance of tokens from the issuer
+* transfer tokens to each other
+* redeem tokens
+* burn tokens
+
+For more information, see the following topics:
+
+* [Installing the Digital Currencies UI]({{< relref "installing-the-dc-demo.md" >}})
+* [Launching the Digital Currencies UI]({{< relref "launching-the-dc-demo.md" >}})
+* [Switching Roles]({{< relref "switching-roles.md" >}})
+* [Working with Token Definitions]({{< relref "defining/_index.md" >}})
+  * [Creating Token Definitions]({{< relref "defining/creating-token-definitions.md" >}})  
+  * [Viewing Token Definitions]({{< relref "defining/viewing-token-definitions.md" >}})
+  * [Activating Token Definitions]({{< relref "defining/activating-token-definitions.md" >}})
+  * [Decommissioning Token Definitions]({{< relref "defining/decommissioning-token-definitions.md" >}})
+* [Minting Tokens]({{< relref "minting/_index.md" >}}) 
+* [Working with Token Issuance]({{< relref "issuing/_index.md" >}})
+  * [Creating Issuance Requests]({{< relref "issuing/creating-issuance-requests.md" >}})
+  * [Viewing Issuance Requests]({{< relref "issuing/viewing-issuance-requests.md" >}})
+  * [Approving or Rejecting Issuance Requests]({{< relref "issuing/approving-or-rejecting-issuance-requests.md" >}})
+  * [Issuing Tokens]({{< relref "issuing/issuing-tokens.md" >}})
+* Transference: 
+    * TBD
```

Comment:
```suggestion
    * TBD
```
Missing title?

Reply:
These topics don't exist yet, will be done in upcoming work.